fedbiomed/transport/server.py

```diff
@@ -298,6 +299,30 @@ async def start(self):
             if self._debug:
                 logger.debug("gRPC server has stopped")
 
+    async def _on_forward(self, message: OverlayMessage) -> None:
```

Comment:
Thank you @srcansiz 
The motivation is to avoid introducing looping calls which seemed to me unwanted design.
We (previously) have `_GrpcAsyncServer` call `AgentStore` call `NodeAgent(NodeAgentAsync)`
As `_on_forward` calls  `_GrpcAsyncServer.send()` this would introduce loop.